src/redux.js

```diff
@@ -114,17 +106,10 @@ function* createSession(action) {
     }
 }
 
-function* signupSlack(action) {
-    try {
-        yield call(waitFor, getSessionCreatedFromState, true)
-        const req = request.post(`${API_URL}/slack/invite`)
-                .send({ email: action.payload.email })
-        const response = yield req
-        yield put({ type: actionTypes.SIGNUP_SLACK_SUCCESS, response })
-    } catch (error) {
-        yield put({ type: actionTypes.SIGNUP_SLACK_FAILURE, error })
-        console.error(error)
-    }
+function* signupDiscord(action) {
+    yield put(actions.signupMailingList(action.payload))
+    yield take(actionTypes.SIGNUP_MAILING_LIST_SUCCESS)
+    yield call(window.open, 'https://discord.gg/wYezN7v')
```

Comment:
Hmmm… Using `window.open` ain't great to be honest. I wonder if popup blockers will kill this 🤔

Do we show any sort of spinner while we're doing the back-and-forth with the backend? If yes, I wonder if we just jump (`window.location`) rather than open a new window?

Reply:
Right, I didn't consider pop-up blockers! Fold actually uses `window.open` as part of the twitter/fb auth flow. 

We don't have a spinner or anything after submit. 😱